phy/scripts/phy_script.py

```diff
@@ -294,11 +294,11 @@ def detect(args):
 
 
 def cluster_auto(args):
+    assert args.file.endswith('.kwik')
```

Comment:
`assert`? this should probably be in `_create_session`, no? And also fail with a more sensible error than just an `AssertError`?

I can imagine some people renaming their .prm to a .kwik ;-)